### DISCUSSION TITLE
Notifications section

### DISCUSSION THREAD

**Discussion by TekyTeky** [Novakasa/brickrail#136] [Ideas]
OK, so I found the handle.. re my comments below.. they are very hard to see/find.
I'll will play around and see how it all goes.. sorry to jump the gun. :-)

Can the notification section start lower and maybe have a pull down too, as it covers the last train controls... even in Fullscreen on a 24 inch monitor.
Although it would be annoying to have to pull it down each time to control trains.
Also what if you have 10 trains?
Would it be better on the Inspector side .. although I know it is linked to the BLE devices.
Or maybe it could be a smaller area and the errors flowed in a scrolling area... with the most recent at the top?
I'm just thinking of options.

<img width="945" alt="Notification panel" src="https://github.com/Novakasa/brickrail/assets/87248517/ab12a22e-6639-4f63-9078-7ac67d006064">

Actually now I have foreced more some errors I see the panel does scroll.. and have popouts.
But maybe needs a sticking handle.. so you can keep it below the last train... but always have the most recent erros visible.
<img width="1154" alt="Notification Panel" src="https://github.com/Novakasa/brickrail/assets/87248517/5cd0ec71-1bc8-48ab-997e-4368e781a639">

**Comment by Novakasa** [comment]
Yeah unfortunately space is getting quite tight now. The section already has a pulldown though, might be hard to find, it is below the "Connect BLE Server" button.

I am currently thinking about what to do about this in the long term. I might move the train controls to the bottom of the screen and make them more compact.

**Reply by Novakasa** [reply]
So, the hubs are still added in the left panel, but controlling trains would be done at the bottom, or in a pop out window

**Reply by TekyTeky** [reply]
May be the trains could be connected on the other side panel with the other info about each of them. Or just have a smaller section for each train on the left.
I think underneath would be ok, and just have the train name and the stop slow, cruise, fast and turn.
They could scroll so if you have a lot, and they could each be in a single line. Then it would be easy to look at the layout and “Play Trains” rather than having to find them in a panel on the left. 
Not sure on a pop out. You don’t want to be fumbling about and having things pop up in front of the main window. Also Color co-ordination is key.  In BAP you can Color the trains like you have, then each train panel (line) is that Color.
What does Rocrail have?

**Reply by TekyTeky** [reply]
Also if you check RocRail, they have each active Loco on a single line with current details.... but their's is for a huge layout and lots of trains.
You could have similar and if you clicked on a train it could popup with more details... or just have its own Inspector panel with all the train details. Then all the controlling at the bottom... in rows also.
So if each train and any other layout items, like switches and crossing etc had their own panel, like you kind of have now on the right. Then it leaves space on the left for the log to run, and also maybe the timetable you plan. AND a clock, like Rocrail, I love the idea of a clock for the timetable.
Sorry just woke up and am excited by the prospects ;-)
